packages/template-retail-react-app/app/pages/checkout/confirmation.test.js

```diff
@@ -212,7 +212,7 @@ afterAll(() => server.close())
 
 test('Navigates to homepage when no order present', async () => {
     renderWithProviders(<Confirmation />, {
-        wrapperProps: {siteAlias: 'uk', locale: 'en-GB'}
+        wrapperProps: {siteAlias: 'uk', locale: {id: 'en-GB'}}
```

Comment:
site and locale are similar, can we make them consistent? either we choose to pass both of them as objects? or we choose to pass them as string (siteAlias and localeAlias)

Reply:
These are implementation details affecting only testing. I don't see it's very important to be consistent here.

We need to specify the `wrapperProps` because we need to write the same tests to pass two scenarios:
1. The Template Retail React App in the monorepo using a Multi-site, Multi-locale configuration.
2. A generated project using the `test-project` with the Singe-site, single-locale configuration.